.gitignore

```diff
@@ -1,5 +1,17 @@
 leelaz
 *.o
 *.d
+.vscode/settings.json
 training/tf/venv
-weights.txt
\ No newline at end of file
+.vs/
+packages/
+*.db
+*.opendb
+*.user
```

Comment:
Do we need these given that the project files are in a subdir? Why not exclude those subdirs instead?

Reply:
Those files are generated by Visual Studio 2015 only, and they are within the msvc folder. So it would be better to have a separate .gitignore file instead.